database/index.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/test')
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
```

Comment:
Maybe use an arrow function here like you did with 'once' below? I did the same thing since it's the boilerplate code from mongoose.